edx_oauth2_provider/oidc/core.py

```diff
@@ -5,16 +5,15 @@
 endpoints according to the OpenID Connect specification.
 
 """
+from __future__ import absolute_import, division, print_function, unicode_literals
 
 import jwt
-
 import provider.scope
+from django.utils.module_loading import import_string
```

Comment:
The code used to import `import_string` from a custom utils.py file, which I deleted, since it was just a copy of the function from a previous Django version. `import_string` is used in the `HANDLERS` def below.